datastream-kafka/src/main/java/com/linkedin/datastream/kafka/KafkaProducerWrapper.java

```diff
@@ -317,7 +316,6 @@ void close(DatastreamTask task) {
         shutdownProducer();
       }
     }
-    ThreadUtils.shutdownExecutor(_producerCloseExecutorService, PRODUCER_CLOSE_EXECUTOR_SHUTDOWN_TIMEOUT, _log);
```

Comment:
Can you open a bug for fixing this and assign it to me? I've sent you an email about it too, so wondering if that approach looks sound? If so, I can take this up after we finish basic low-level certification.